README.md

```diff
@@ -100,6 +101,43 @@ Every time this app receives an update you should restart the systemd service us
 
 `sudo systemctl restart notify_push`
 
+Alternatively, you can do this automatically via systemctl by creating the following systemd service and path:
+
+`/etc/systemd/system/notify_push-watcher.service`
+```ini
+[Unit]
+Description=Restart Push daemon for Nextcloud clients when it receives updates
+Documentation=https://github.com/nextcloud/notify_push
+Requires=notify_push.service
+After=notify_push.service
+StartLimitIntervalSec=10
+StartLimitBurst=5
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/systemctl restart notify_push.service
+
+[Install]
+WantedBy=multi-user.target
+```
+
+`/etc/systemd/system/notify_push-watcher.path`
+```ini
+[Unit]
+Description=Restart Push daemon for Nextcloud clients when it receives updates
+Documentation=https://github.com/nextcloud/notify_push
+PartOf=notify_push-watcher.service
+
+[Path]
+PathModified=/path/to/push/binary/notify_push
+Unit=notify_push-watcher.service
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Adjusting the path as needed.
+
```

Comment:
```suggestion
And enable it with:
```bash
systemctl enable --now notify_push-watcher.path
``` <- formatig error of github
```

---
You should something write, like my suggestion - to inform which of the systemd unit's should be started (.path or .service)